.github/workflows/main.yml

```
@@ -0,0 +1,743 @@
+      run: |
+        git fetch origin main
+        GITHUB_COMPARE_BRANCH="origin/main"
+        if [ "${{ github.ref_name }}" == "main" ]; then
+          GITHUB_COMPARE_BRANCH="${{ env.COMMIT_FULL_ID }}^"
+        fi
+        get_changed_charts=`git diff ${{ env.COMMIT_FULL_ID }} "$GITHUB_COMPARE_BRANCH" --name-only | grep -oE "^kubernetes/(\w+[-]?)+(\w+)" | grep -oE "(\w+[-]?)+(\w+)$" | grep -v README| sort -u`
+        changed_charts=$(echo $get_changed_charts | tr " " ",")
+        changed_charts="$(echo -e "$changed_charts" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+        echo "Changed charts: $changed_charts"
+        echo "changed_charts=$changed_charts" >> $GITHUB_OUTPUT
+    - name: Checkout stacked_charts 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)